packages/compartment-mapper/src/bundle.js

```diff
@@ -11,6 +13,42 @@
 /** @import {Sources} from './types.js' */
 /** @import {WriteFn} from './types.js' */
 
+/**
+ * @typedef {object} BundlerKit
+ * @property {() => string} getFunctor Produces a JavaScript string consisting of
+ * a function expression followed by a comma delimiter that will be evaluated in
+ * a lexical scope with no free variables except the globals.
+ * In the generated bundle runtime, the function will receive an environment
+ * record: a record mapping every name of the corresponding module's internal
+ * namespace to a "cell" it can use to get, set, or observe the linked
+ * variable.
+ * @property {() => string} getCells Produces a JavaScript string consisting of
+ * a JavaScript object and a trailing comma.
+ * The string is evaluated in a lexical context with a `cell` maker, the `cells`
+ * array of every module's internal environment record.
+ * @property {() => string} getFunctorCall Produces a JavaScript string with a
+ * single statement that will call this module's functor with the calling
```

Comment:
The implementation in this file returns a string consisting only of a comment, which therefore does not "call this module's functor"